src/Http/Controllers/V1/Shop/Customer/CartController.php

```diff
@@ -22,7 +22,7 @@ public function get()
         $cart = Cart::getCart();
 
         return response([
-            'data' => $cart ? new CartResource($cart) : null,
+            'data' => $cart ? new CartResource($cart) : __('rest-api::app.checkout.cart.item.no-item'),
```

Comment:
Refer to here,
https://github.com/bagisto/rest-api/pull/47#discussion_r1161878446